Streamline Setup And Render Process

sohrabhamza opened the pull request:
Added some options to the setup process to accommodate people who do not use Google Drive for their blend file while not affecting those who do
Added some options to the Render Process to make it easier to render all frames, one specific frame, or frame in between a range.

This is my first time using github so please forgive any mistakes.

sohrabhamza committed:
Streamline Setup And Render Process

Added some options to the setup process to accommodate people who do not use Google Drive for their blend file while not affecting those who do
Added some options to the Render Process to make it easier to render all frames, one specific frame, or frame in between a range.

This is my first time using github so please forgive any mistakes.

donmahallem commented on runblender.ipynb:
```diff
@@ -31,30 +31,32 @@
-        "#@markdown ---\n"
+        "use_drive = False #@param {type:\"boolean\"}\n",
+        "#@markdown ---\n",
+        "if use_drive:\n",
+        "  from google.colab import drive\n",
+        "  drive.mount('/content/drive')\n",
```
I would seperate this still into a seperate code cell. So configuration and execution are distinct

sohrabhamza replied:
Sure

donmahallem commented on runblender.ipynb:
```diff
@@ -196,28 +187,13 @@
-        "drive.mount('/gdrive')"
-      ],
-      "execution_count": 0,
+      "execution_count": 8,
```
Could you clear the execution_count across the document and set them to 0 or null?

sohrabhamza replied:
Ok

donmahallem reviewed (changes requested):
Looks good to me! Before I merge could you reduce the noise in changes lines?

sohrabhamza commented:
This is my first time using github so this is quite confusing to me. 

>Looks good to me! Before I merge could you reduce the noise in changes lines?

How do I do this?

donmahallem commented:
I will try to handle it then.

sohrabhamza committed:
Update Based on Requirement

Separated the drive init. Made the execution_count = null
I did not understand what the line noise meant so didn't do anything about that.

sohrabhamza commented:
Removed the drive init from setup and made execution_count = null

donmahallem committed:
Update runblender.ipynb

sohrabhamza commented:
Cool!
Can I now close the pull request?
What do I do with the fork?

donmahallem commented:
I believe I fixed up everything. Would you mind checking my changes and report back ? I could merge your proposals if you are okay with it?

donmahallem commented:
> Cool!
> Can I now close the pull request?
> What do I do with the fork?

Once I merged this Pull Request you could delete the fork. Did you check my changes to your Pull Request if they are working and working as you intended?

sohrabhamza commented:
Yeah I checked. Looks fine.

sohrabhamza commented:
Thanks!
